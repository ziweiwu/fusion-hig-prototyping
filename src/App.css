@import "~@hig/styles/build/index.css";

.App {
  font-family: ArtifaktElement;
  padding-bottom: 25px;
}

body {
  width: 1054px;
  height: 1054px;
}
.side-area {
  background-color: #0696D7;
  padding-bottom: 25px;
}
