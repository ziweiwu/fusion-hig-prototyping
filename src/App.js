import React, {Component} from 'react';
import './App.css';
import "@hig/styles/build/index.css";
import DatePicker from "./components/myDatePicker/index";
import moment from "moment"

class App extends Component {
  render() {
    return (
      <div className="App">
        <h1>App Started </h1>
        <DatePicker startTime={moment("2017-09-15 09:30:00")}/>
      </div>
    );
  }
}

export default App;
