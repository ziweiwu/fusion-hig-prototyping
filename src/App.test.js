import App from './App';
import React from 'react';
import { mount, shallow } from 'enzyme';

describe('App', () => {
  // snapshot testing
  test('App snapshot matches', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  // component testing
  it('Render the App component correctly.', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find('h1')).toHaveLength(1);
  });
});
