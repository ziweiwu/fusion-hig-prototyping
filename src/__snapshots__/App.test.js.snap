// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot matches 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="App"
>
    <h1>
        App Started 
    </h1>
    <DatePicker
        startTime={"2017-09-15T13:30:00.000Z"}
    />
</div>,
  "nodes": Array [
    <div
      className="App"
>
      <h1>
            App Started 
      </h1>
      <DatePicker
            startTime={"2017-09-15T13:30:00.000Z"}
      />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <App />,
    "_forcedUpdate": false,
    "_instance": App {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": null,
      "updater": Updater {
        "_callbacks": Array [],
        "_renderer": [Circular],
      },
    },
    "_newState": null,
    "_rendered": <div
      className="App"
>
      <h1>
            App Started 
      </h1>
      <DatePicker
            startTime={"2017-09-15T13:30:00.000Z"}
      />
</div>,
    "_rendering": false,
    "_updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
